### PR TITLE
chore: e2e with maestro

### PR DESCRIPTION
**Pull request by vanGalilea**
## Goal:
This PR aims to introduce, configure and make E2E accessible for all of our engineers.

## Next:
- Running e2e in CI/CD when a PR after a PR is being created
- Converting run test shell script to fastlane file
- Moving meet starter kit to samples folder

### CODE REVIEW

File: packages/react-native-sdk/src/components/LocalVideoView.tsx [hunk 1]
```diff
@@ -1,5 +1,12 @@
-import React from 'react';
```
Comment (vanGalilea) [inline packages/react-native-sdk/src/components/LocalVideoView.tsx:1]: This code is already in main and doesn't need review, GH is wrongfully showing it

**Review by testableapple — APPROVED** [review]
LGTM! 🚢

File: packages/react-native-meeting-starter-kit/README.md [hunk 2]
```diff
@@ -28,3 +28,12 @@ The following are the installation steps to run the application:
 - Move to `cd packages/react-native-meeting-starter-kit`.
 - Pod install for iOS using `npx pod-install` or `cd ios && pod install`.
 - Run `yarn start` and `yarn run android` to run the application on Android, and, `yarn run ios` to run it on iOS.
+
+### Running the E2E tests
```
[the remark this answers is not in the document]
Comment (vanGalilea) [reply packages/react-native-meeting-starter-kit/README.md:32]: Please read about [Maestro](https://maestro.mobile.dev/) and it's [studio](https://maestro.mobile.dev/getting-started/maestro-studio) that makes our lives easier

[the remark this answers is not in the document]
Comment (vanGalilea) [reply packages/react-native-meeting-starter-kit/README.md:32]: @khushal87 and @santhoshvai ^^

**Pull request merged**